### PR TITLE
lint: clean up un-linted js files

### PR DESCRIPTION
**Pull request by mvadari**
## High Level Overview of Change

This PR lints `src/registerServiceWorker.js` and `src/setupTests.js`, which somehow aren't in the default eslint/prettier path.

### Context of Change

The files weren't linted, and it made some diffs larger than needed.

### Type of Change

- [x] Refactor (non-breaking change that only restructures code)

## Before / After

N/A

## Test Plan

CI passes.

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 build/
 coverage/
 node_modules/
+src/registerServiceWorker.js
```
Comment (ckniffen) [inline .eslintignore:4]: Why is this file being ignored?

File: package.json [hunk 2]
```diff
@@ -224,6 +224,21 @@
...
+        "error",
+        {
+          "devDependencies": [
+            "**/*.test.js",
+            "**/*.test.jsx",
+            "**/*.test.ts",
+            "**/*.test.tsx",
+            "**/test/**",
+            "**/setupTests.ts",
+            "**/svgTransform.js",
```
Comment (ckniffen) [inline package.json:238]: Is this a file added in the vite work?

File: .eslintignore (hunk 1 above)
Comment (mvadari) [reply .eslintignore:4]: There are a bunch of errors and I didn't really know how to resolve them

**Review by ckniffen — APPROVED** [review]

**Review by jonathanlei — APPROVED** [review]
LGTM, should double check if it works with react-17

**Pull request merged**